src/test/java/co/omise/requests/CustomerRequestTest.java

```diff
@@ -51,8 +42,11 @@ public void testCreate() throws IOException, OmiseException {
 
     @Test
     public void testUpdate() throws IOException, OmiseException {
-        Customer customer = resource().update(CUSTOMER_ID, new Customer.Update()
-                .email("john.doe.the.second@example.com"));
+        Request<Customer> request = new Customer.UpdateRequestBuilder(CUSTOMER_ID)
+                .email("john.doe.the.second@example.com")
+                .build();
+        Customer customer = getTestRequester().sendRequest(request);
+
         assertRequested("PATCH", "/customers/" + CUSTOMER_ID, 200);
 
         assertEquals("cust_test_4yq6txdpfadhbaqnwp3", customer.getId());
```

Comment:
Is this `cust_test_4yq6txdpfadhbaqnwp3` same as `CUSTOMER_ID`?

Reply:
Yes should have used `CUSTOMER_ID`, will make changes